projects/developer_tooling/developer_tooling.md

```diff
@@ -1,47 +1,48 @@
-## Project Name <!-- Add your project name here"-->
-<!-- Add you project logo here if available-->
+## Project Name 
+Filecoin Flavored Ganache 
 
 ## Category 
-<!--developer tooling, application, wallet, infrastructure, etc-->
+Developer Tooling
 
 ## Project Description
-<!--Describe your project in a few sentences. -->
+Ganache was built as the sister application to Truffle, to speed up development and testing Ethereum applications. Now that we've added support to Truffle, it's only natural we provide the same highly-modular simulator for Filecoin. We are currently creating a Filecoin simulation library, as part of ganache-core. We are also developing support in our graphical version of Ganache. You can checkout some early mocks under previews!
 
 ## Project Status
-<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+Alpha version released & stable version under development
 
 ## Previews
-<!--Add some screenshots to give a preview of your product-->
+https://drive.google.com/file/d/1C3gRnm0-aF1xGgv5EcNRuvkpAQrhLVID/view?usp=sharing
 
 ## Target Audience
-<!--Describe who will be your project's users-->
+Developers
 
 ## Rough estimated user base (if applicable)
-<!--How many users do you have right now?-->
+Truffle Suite has about 6 million downloads across our suite of tools including Truffle, Ganache, Drizzle, and Teams
 
 ## Github repo
-<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/trufflesuite
+Alpha version of Filecoin Flavored Ganache is available here: https://github.com/trufflesuite/ganache-filecoin-alpha-cli
 
 ## Website
-<!--Link your website if available-->
+https://www.trufflesuite.com/
 
 ## Docs
-<!--Including a link to your project docs!-->
+Our docs are located here: https://www.trufflesuite.com/docs
+More docs for this project will be published in April 2021.
 
 ## Team Info
-<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+
 ### Team Size  
+Truffle Suite is a team of 13 developers working across our sweet suite of tools.
 
-### Team members  
+### Team members
+The main developers on this projects are @tcoulter and @seesemichaelj.
 
 ## How the community can engage
-GitHub Discussion: <!--Start a disucssion with the community here: https://github.com/filecoin-project/community/discussions/new and attach the link!-->  
-Email:  
-Slack:  
-Twitter:  
-Discord:  
-Telegram:  
-WeChat:  
+Email: inquiry@trufflesuite.com
```

Comment:
Please create a project discussion, with the category as Show and tell - Developers and title as [Category] Project Name(i.e: [Application] Slate), link the discussion in the How the community can engage section. This is to create a place to interact with the Filecoin community, share your updates and gain feedback on your projects!

Reply:
Updated! Also opted into the newsletter